src/js/entities-service/entities/clinicians.js

```diff
@@ -25,7 +25,7 @@ const _Model = BaseModel.extend({
     return Radio.request('entities', 'groups:collection', this.get('_groups'));
   },
   getRole() {
-    if (!this.get('_role')) {
+    if (!this.get('_role') && !this.isNew()) {
```

Comment:
Added this to allow for an empty role on a new clinician model so that we can display the droplist default text of "Select Role..."